clients/client-python/gravitino/api/expressions/distributions/distribution.py

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from abc import abstractmethod
+from typing import List
+
+from gravitino.api.expressions.distributions.strategy import Strategy
+from gravitino.api.expressions.expression import Expression
+
+
+class Distribution(Expression):
+    """
+    An interface that defines how data is distributed across partitions.
+    """
+
+    @abstractmethod
+    def strategy(self) -> Strategy:
+        """Return the distribution strategy name."""
+
+    @abstractmethod
+    def number(self) -> int:
```

Comment:
```suggestion
    def buckets(self) -> int:
```

Reply:
Do we need to make the name consist  to the java file?

Reply:
@yuqi1129 Can you give some suggestions?

Reply:
Both are acceptable to me, I would prefer to keep it consistent with that in Java.

Reply:
If we need to modify, I think we can do this in another PR to modify function name in the Python and Java client at same time.